templates/vue/package.json

```diff
@@ -19,12 +19,14 @@
     "babel-polyfill": "^6.26.0",
     "bootstrap": "^4.3.1",
     "bootstrap-vue": "^2.13.1",
+    "jquery": "^1.9.1",
     "moment": "^2.25.3",
     "moment-timezone": "^0.5.28",
     "vue": "^2.6.10",
     "vue-router": "^3.1.3",
     "vue-slicksort": "^1.1.3",
-    "vuex": "^3.1.1"
+    "vuex": "^3.1.1",
+    "watchpack-chokidar2": "^2.0.0"
```

Comment:
What is this package for?

Reply:
I believe this came in when I attempted to update fsevents, which I couldn't/ didn't need to update anyway.